External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -60,18 +63,36 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 	end
 end
 
+if nargin < 7 || isempty(bSaveJson)
+    bSaveJson = false;
+end
+
+if nargin < 8

```

Comment:
I understand that you didn't add isempty. But actually, we should add || isempty as well. Cause if somebody provides [], then we should still replace that by an empty struct.

Reply:
Done in